weather_dl/download_pipeline/pipeline.py

```diff
@@ -180,6 +181,10 @@ def run(argv: t.List[str], save_main_session: bool = True) -> PipelineArgs:
 
     known_args, pipeline_args = parser.parse_known_args(argv[1:])
 
+    if "DataflowRunner" in pipeline_args and  "--sdk_container_image" not in pipeline_args:
```

Comment:
I think this should go in the `weather-dl` script.

Reply:
Updated.